spec/acceptance/workflow_regen_after_expire_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 # https://forge.puppet.com/puppetlabs/certregen#revive-a-ca-thats-already-expired
 describe "C99821 - workflow - regen CA after it expires" do
-  if hosts_with_role(hosts, 'master').length>0 and hosts_with_role(hosts, 'database').length>0 then
+  if find_install_type == 'pe' then
```

Comment:
Do PE tests imply that these roles are present, and do FOSS tests never exercise this code? The new code seems to be a bit orthogonal with respect to the old code.